troposphere/firehose.py

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) 2014, Guillem Anguera <ganguera@gmail.com>
```

Comment:
If you don't care could you switch it to (for lack of a better one):
# Copyright (c) 2016, troposphere project

I have no plans to ever change the licensing but I do care about keeping copyrights intact in case there is any issues. Thanks!